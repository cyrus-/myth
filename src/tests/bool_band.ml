let s = Bool_decls.s ^ "

let bool_band : bool -> bool -> bool |>
  { True => True => True
  ; True => False => False
  ; False => True => False
  ; False => False => False } = ?"
