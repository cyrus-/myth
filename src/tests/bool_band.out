(* Generated Thu Nov 13 00:35:22 2014 *)
let bool_band : bool -> bool -> bool =
  fun (b1:bool) -> fun (b2:bool) -> match b1 with
                                      | True -> b2
                                      | False -> False
;;
